mm2_install.sh

```diff
@@ -33,6 +32,7 @@ sudo chown $username:$username ~/modesmixer2
 
 echo "copy czadsb starter script"
 sudo cp ~/czadsb-scripts/starters/czadsb.modesmixer.sh ~/modesmixer2/czadsb.modesmixer.sh
+sed -i 's/USER/'$username'/g' ~/modesmixer2/czadsb.modesmixer.sh
```

Comment:
toto je špatně:
v tom souboru `czadsb.modesmixer.sh` je `/home/USERNAME/modesmixer....` - takže když nahradíš jen `USER` tak je výsledek: `piNAME`

Reply:
Diky. zmenil jsem to na s/USERNAME uz by to melo fungovat.